tests/test_operators.py

```diff
@@ -746,3 +746,130 @@ def test_free_operator() -> None:
             ]
         )
     )
+
+
+def test_free_conversion_operator() -> None:
```

Comment:
This is generated by script (the comment at the top of the file was very useful!)